notebooks/ch14/bias_modeling.md

```diff
@@ -57,7 +57,7 @@ R(f_\hat{\theta}) = \mathbb{E}[ \ell(\gamma, f_\hat{\theta} (z)) ]
 \end{aligned}
 $$
 
-We denote the process that generates the true population data as $ f_\theta $. Our observed outcomes $ y $ were generated by our population process plus some random noise in data collection: $ y_i = f_\theta(X_i) + \epsilon $, where the random noise $ \epsilon $ is a random variable with a mean of zero: $ \mathbb{E}[\epsilon] = 0 $.
+We denote the process that generates the true population data as $ f_\theta(x) $. Our observed outcomes $ \gamma $ were generated by our population process plus some random noise in data collection: $ \gamma_i = f_\theta(z_i) + \epsilon $.  The random noise $ \epsilon $ is a random variable with a mean of zero: $ \mathbb{E}[\epsilon] = 0 $.
```

Comment:
nit: \gamma doesn't represent the observed outcomes (since y represents the observed outcomes), it represents the outcome from the population.

We should instead say:

> The output point $ \gamma $ is generated by ...